app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -2696,6 +2699,29 @@ export class Dispatcher {
     this.cherryPick(repository, targetBranch, commits, sourceBranch)
   }
 
+  /**
+   * Method to start a cherry-pick after drag and dropping onto a pull request.
+   */
+  public async startCherryPickWithPullRequest(
+    repository: RepositoryWithGitHubRepository,
+    pullRequest: PullRequest
+  ) {
+    const targetBranch = await this.appStore._findPullRequestBranch(
+      repository,
+      pullRequest
+    )
```

Comment:
Related to what I ask in https://github.com/desktop/desktop/pull/11921#discussion_r606976290 maybe we can reuse the logic of "checkout to PR", grab the resulting branch from that operation, and use that as `targetBranch`.